src/configuration/services/memcached.md

```diff
@@ -8,7 +8,11 @@ Both Memcached and Redis can be used for application caching.  As a general rule
 
 ## Supported versions
 
-* 1.4
+<div id = "memcachedSupported"></div>
+
+<script>
+makeImagesList("services", "memcached", "supported", "memcachedSupported");
```

Comment:
I believe we can reduce it to just
```
makeImagesList("services", "memcached");
```

Why do we have a list of memcached version that we don't support?

The same thing to the other services.

Reply:
> I believe we can reduce it to just
>
> `makeImagesList("services", "memcached");`

For this function no. "supported"/"deprecated" pulls those versions from `versions.json`, and "memcachedSupported" identifies the `div id`.

> Why do we have a list of memcached version that we don't support?

We already document deprecated versions we do not support in the documentation, so this is following the same formatting. It make sense for those to be there in cases where a user is migrating a pre-existing application to Platform, but maybe using an older version of a service/language. It's a heads up that says "beyond just configuring for Platform, you'll have to upgrade some stuff first."